libcontainer/utils/utils_unix.go

```diff
@@ -98,3 +100,55 @@ func NewSockPair(name string) (parent, child *os.File, err error) {
 	}
 	return os.NewFile(uintptr(fds[1]), name+"-p"), os.NewFile(uintptr(fds[0]), name+"-c"), nil
 }
+
+// ToSchedAttr is to convert *configs.Scheduler to *unix.SchedAttr.
+func ToSchedAttr(scheduler *configs.Scheduler) *unix.SchedAttr {
+	var policy uint32
+	switch scheduler.Policy {
+	case specs.SchedOther:
+		policy = 0
+	case specs.SchedFIFO:
+		policy = 1
+	case specs.SchedRR:
+		policy = 2
+	case specs.SchedBatch:
+		policy = 3
+	case specs.SchedISO:
+		policy = 4
+	case specs.SchedIdle:
+		policy = 5
+	case specs.SchedDeadline:
+		policy = 6
+	}
```

Comment:
Should we return an error in case Policy is set to some unknown value?

Reply:
I have considered it in the mind, but I thought that because the default value is `SchedOther` in the kernel, so maybe we should always use `SchedOther`(policy=0) by default.
Maybe we should return an error for some unknown values(**except a blank string**) because there is no check for `Policy` value in validator.